packages/blockchain-wallet-v4-frontend/src/layouts/Nfts/NftsHeader/NftsSearch/NftsSearch.tsx

```diff
@@ -100,9 +102,14 @@ const NftsSearch: React.FC<Props> = ({ nftActions, nftSearch, routerActions }) =
     setIsActive(true)
   }
 
+  const search = useCallback(
+    debounce((input) => nftActions.nftSearch({ search: input }), 500),
+    []
```

Comment:
this should have the input, right? Otherwise it won't have the value from the last render

Reply:
I just want the original debounce call in the last .5 seconds.  If I add input, it'll keep making new debounce functions